app/backend/controllers/login-signup.js

```diff
@@ -110,6 +151,9 @@ module.exports.forgetPassword = async (request, response) => {
         throw Error('User not found.')
       }
 
+      if(userRegistered.googleId)   // if user registered via Google, he/she cannot change password
```

Comment:
Using curly braces even if it would contain only one line of code is highly recommended.